src/main/java/net/mcreator/minecraft/link/CurrentDevice.java

```diff
@@ -147,14 +147,23 @@ public static void disableDigitalInputEvents(int port) {
 	 * <p>
 	 * If there is no device connected, nothing happens.
 	 *
-	 * @param message The message to be sent
+	 * @param command The message to be sent
 	 */
-	public static void sendMessage(String message) {
+	public static void sendMessage(String command) {
 		AbstractDevice device = MCreatorLink.LINK.getConnectedDevice();
+		System.out.println("pure message: " + command);
```

Comment:
Remove debug prints

Reply:
This still needs removal

Reply:
Yeah, I did it, but in reverted commit they came back and I didn't notice it
Done already.